File: Net-Core-API/Models/Student.cs
using System.ComponentModel.DataAnnotations;
namespace Net_Core_API.Context
{
    public class Student
    {
        [Key]
        /// <summary>
        /// Student Id
        /// </summary>
        public int Id { get; set; }
        /// <summary>
        /// Student user name
        /// </summary>
        public string UserName { get; set; }
        /// <summary>
        /// Student first name
        /// </summary>
        /// <example>FirstNameLastName</example>
        public string FirstName { get; set; }
        /// <summary>
        /// Student last name
        /// </summary>
        public string LastName { get; set; }
        /// <summary>
        /// Student age
        /// </summary>
        public int Age { get; set; }
        /// <summary>
        /// Student career
        /// </summary>
        public string Career { get; set; }
    }
}

File: Net-Core-Front/src/app/models/Student.ts
export interface IStudent {
    id: number;
    userName: string;
    firstName: string;
    lastName: string;
    age: number;
    career: string;
}

export class Student {
    public id: number;
    public userName: string;
    public firstName: string;
    public lastName: string;
    public age: number;
    public career: string;
    constructor() {
        this.id = null;
        this.userName = "";
        this.firstName = "";
        this.age = 0;
        this.career = "";
    }
}
